Firestore/core/src/firebase/firestore/model/transform_operations.h

```diff
@@ -97,9 +97,8 @@ class ServerTimestampTransform : public TransformOperation {
   FSTFieldValue* ApplyToLocalView(
       FSTFieldValue* previousValue,
       const Timestamp& local_write_time) const override {
-    return [FSTServerTimestampValue
-        serverTimestampValueWithLocalWriteTime:local_write_time
-                                 previousValue:previousValue];
+    return FieldValue::FromServerTimestamp(local_write_time, previousValue)
```

Comment:
Optional nit: s/previousValue/previous_value.

Reply:
I did this in #3178 (a later PR)